controllers/confidential_handler.go

```diff
@@ -0,0 +1,132 @@
+package controllers
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func (r *KataConfigOpenShiftReconciler) updateConfigMap(cmName string, namespace string, newData map[string]string) error {
```

Comment:
@spotlesstofu would it make sense to move this method to utils.go as commonly used methods are kept in that file and this method could be useful for other featuregates as well ?